README.md

```diff
@@ -1,9 +1,81 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/node-driver-registrar.svg?branch=master)](https://travis-ci.org/kubernetes-csi/node-driver-registrar)
 # Node Driver Registrar
+Latest stable image: `quay.io/k8scsi/csi-node-driver-registrar:v1.0.1`
 
-A sidecar container that
+The node-driver-registrar is a sidecar container that registers the CSI driver using the
+[kubelet plugin registration mechanism](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#device-plugin-registration).
```

Comment:
How about:

> The `node-driver-registrar` is a sidecar container that registers the CSI driver with Kubelet using the [kubelet plugin registration mechanism](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#device-plugin-registration). 
> 
> This is necessary because Kubelet is responsible for issuing CSI `NodeGetInfo`, `NodeStageVolume`, and `NodePublishVolume` calls. In order to figure out which Unix domain socket to issue the command(s) on, Kubelet checks an internal table. The `node-driver-registrar` registers your CSI driver with Kubelet so that the unix domain socket for your driver is available to kubelet.

Reply:
done with some modifications to the wording